build.sbt

```diff
@@ -327,13 +316,10 @@ lazy val compileInterfaceProj = (project in compilePath / "interface").
     // needed because we fork tests and tests are ran in parallel so we have multiple Scala
     // compiler instances that are memory hungry
     javaOptions in Test += "-Xmx1G",
-    artifact in (Compile, packageSrc) := Artifact(srcID).copy(configurations = Compile :: Nil).extra("e:component" -> srcID)
+    publishArtifact in (Compile, packageSrc) := true,
+    publishMavenStyle := true
```

Comment:
To make it Maven friendly, we should sync to Maven central, and thus far we've only taken those steps after we split subproject out to a repository, like sbt/launcher and sbt/io. So I think we think we should wait for it until we have a separate Compiler API, which hopefully we'll have soon.